returnn/extract_prior.py

```diff
@@ -114,12 +132,11 @@ def __init__(
             if returnn_python_exe is not None
             else gs.RETURNN_PYTHON_EXE
```

Comment:
both `gs.` calls need to be wrapped with `tk.Path`, otherwise later code will not work.